### PR TITLE
Add minio to test builds locally

### PR DESCRIPTION
**Pull request by NyakudyaA**
* Add minio to facilitate image setup on local build

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 3 sec

File: .example.env [hunk 1]
```diff
@@ -12,12 +12,12 @@ DATABASE_PORT=5432
 SERVER_MODE=production
 RECAPTCHA_PUBLIC_KEY=
 RECAPTCHA_PRIVATE_KEY=
-SMTP_EMAIL=
-SMTP_EMAIL_TLS=
 SMTP_HOST=
-SMTP_HOST_PASSWORD=
+SMTP_PORT=587
```
Comment (tinashechiraya) [inline .example.env:16]: This value won't need to be hardcoded since it will be assigned on staging

File: .example.env [hunk 2]
```diff
@@ -42,3 +42,8 @@ SMTP_HOST=
 SMTP_HOST_PASSWORD=
 SMTP_HOST_USER=
 SMTP_PORT=
+
+# Minio
```
Comment (tinashechiraya) [inline .example.env:46]: these values will also be dynamic on staging ,so I'm assuming these are defaults for local?

**Review by tinashechiraya — APPROVED** [review]

File: .example.env (hunk 2 above)
Comment (NyakudyaA) [reply .example.env:46]: Currently staging and production are using the same values, these are just for local development

File: .example.env (hunk 1 above)
Comment (NyakudyaA) [reply .example.env:16]: This is just for local, settings for dev and staging are defined in the devops repo

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 1 sec

**Pull request merged**